Tests/CommandsTests/TestCommandTests.swift

```diff
@@ -159,6 +159,7 @@ struct TestCommandTests {
             }
         } when: {
             (buildSystem == .swiftbuild && ProcessInfo.hostOperatingSystem == .windows)
+            || (buildSystem == .swiftbuild && ProcessInfo.hostOperatingSystem == .macOS)
```

Comment:
issue: this seems like a regression in behaviour since this is now failing on macOS, where it was previously passing..  Is there an issue tracking the proper resolution?